elasticsearch/init/0100-seed-pipelines

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -e
+#
+# Copyright 2018 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "logging"
+
+info Adding ingest pipelines
+shopt -s failglob
+for ingest_file in ${ES_HOME}/ingest_pipelines/*.json
+do
+    if [ -f ${ingest_file}.bkup ]; then
+      rm ${ingest_file}
+      mv ${ingest_file}.bkup ${ingest_file}
+    fi
```

Comment:
we shouldn't need this do we? we aren't doing any `sed` operations...

Reply:
We do not. I will remove it and we can add later if needed.